app/controllers/targets_controller.rb

```diff
@@ -45,6 +47,7 @@ def matches
 
   def create_fail(message)
     flash.now[:alert] = message
+    new
```

Comment:
just calling new doesn't end up doing the render :new ? or maybe a redirect could work?

Reply:
I tried with; only new, only render :new, only redirect, and none of them did the job.